### ISSUE TITLE
celestialObject size for Earth 

### ISSUE THREAD

**Issue by ROMVoid95** [LemADEC/WarpDrive#414]
**Describe the bug**
When the mod is active on a server, the default size for the "Earth" / Overworld is 5000x5000 per the celestialObjects-default.xml. This configuration seems to override any other server settings, or plugins if running Sponge as well (Nucleus, WorldBorder, etc...) i've tested with/without sponge, and with/without warp drive installed. 

However, even changing the size higher than its default value still keeps the world border at 5000x5000. 

**Expected behavior**
Possibly have the mod detect the size of the overworld vs setting the size in the celestialObjects-default.xml. 

**Logs**
no logs, but heres the list of mods/plugins 
https://hastebin.com/avehogisin.coffeescript

**Versions** (latest is not a valid answer)
 - WarpDrive 1.12.2-1.5.11.SNAPSHOT
 - Forge 14.23.5.2847
 - Minecraft 1.12.2
 - SpongeForge 1.12.2-2838-7.1.7-RC3928

**Comment by LemADEC** [comment]
The WarpDrive world border lives independently from the vanilla one.
There's multiple reasons for that:
1- Vanilla only has a single world information (including border). Because of this Vanilla design, you can't have different border per dimension, which doesn't work with a space dimension covering multiple "planet" dimensions. Many mods have similar issues and they are changing the base game to split it, allowing different world information for each dimension: Mystcraft, Sponge, OpenTerrainGenerator, JustEnoughDimensions, just to name a few. Those mods are hardly compatible with each others, so it's easier to maintain an overlaying border than try to be compatible with them.
2- CelestialObjects are designed to allow more than 32k different planets. To this effect, they were designed initially to have multiple planets inside the same dimension, which implies to have a different world border depending on your location. This was tested on 1.7.10, and in theory still works on 1.12.2. This imply that WarpDrive can't use the vanilla WorldBorder logic.

As a consequence of 1-, the vanilla World Border needs to be bigger than Hyperspace. If you see the vanilla world border while you're not in hyperspace, you'll die from it when going to space or hyperspace.
In that case, you can use vanilla commands to restore to its default, see https://minecraft.gamepedia.com/World_border#Commands .

I've only seen this happens when adding/removing a mod that changed how world information is handled. I've never seen it from changing CelestialObjects sizes.

Long story short, please provide exact steps to reproduce your issue so we can understand what's going on here.

**Comment by LemADEC** [comment]
No update, assuming fixed

**Comment by LemADEC** [comment]
Next update will maximize the WorldBorder when loading a save. That's just a workaround until we can figure out which plugin/mod is corrupting it.
